web/src/js/components/common/MonacoEditor.jsx

```diff
@@ -0,0 +1,34 @@
+//not working
```

Comment:
@mhils Yes, but I think we can use `react-ace` or `react-codemirror` instead, any reason for using monaco here?